### DISCUSSION TITLE
Exploring Collaboration with Akash Network for HackArcode 2024 Hackathon

### DISCUSSION THREAD

**Discussion by abhinavcse22** [akash-network/community#645] [Governance Proposals]
Hello Akash Team,

I'm reaching out to propose a partnership between Akash Network and HackArCode, a 24-hour hackathon scheduled for September 7-8, 2024, at IIIT Delhi, India. Our event is designed to bring together over 300 developers, tech enthusiasts, and professionals to work on innovative projects and tackle real-world challenges within a short timeframe.

Given Akash Network's leading position in decentralized cloud computing, we believe a partnership could offer significant benefits to both parties. HackArCode aims to foster creativity and collaboration, and we see Akash's unique approach to peer-to-peer computing resources as a perfect match for our hackathon.

Here's how we envision Akash Network's involvement in HackArCode:

1. Sponsorship:
   - Akash can sponsor our hackathon, providing resources and financial support to help us deliver a successful event.

2. Workshops and Mentoring:
   - We invite Akash experts to conduct workshops on decentralized cloud computing and to mentor participants during the hackathon.

3. Project Evaluation:
   - Akash representatives can join our judging panel, providing expert feedback to participants and evaluating their projects.

These elements align with Akash's commitment to innovation and community building. By engaging with our hackathon, Akash can connect with a vibrant group of developers and tech enthusiasts, and further promote its decentralized cloud platform.

Previous Hackathons Organized:

1. Hackwithindia:
   - Registration Count: 5000+
   - Participation: 1500+ in the online round and 200+ in the final offline round

2. Hacknovate 5.0:
   - Registration Count: 3000+
   - Participation: 350+ participants

3. BuildwithDelhi:
   - Registration Count: 2000+
   - Participation: 200+ participants

We'd be delighted to discuss the details of this potential partnership and explore how Akash Network can contribute to HackArCode's success. Please let me know if we can schedule a call to discuss further.

Thank you for your consideration, and I look forward to hearing from you soon.

Best regards,  
Abhinav Gupta  
Organizer HackArCode
Email: arcodehack@gmail.com  

These Are our Social Media Handles: 

LinkedIn: https://www.linkedin.com/company/hackarcode/
Twitter: https://x.com/hackarcode
Discord: https://bit.ly/hackarcode

**Comment by hewi333** [comment]
Hey @abhinavcse22 thank you for bringing this opportunity forward. Do you have an amount of funding you are seeking or just support?

**Reply by abhinavcse22** [reply]
Hi @heyhewi  ,

Thank you for considering our sponsorship proposal for HackArCode. We are looking for a sponsorship amount of 1000 AKT to support the event. However, we are open to discussing this amount to reach a figure that works for both sides. Your support will be crucial for the success of the hackathon, and Akash Network will get significant visibility and the chance to connect with a vibrant group of developers and tech enthusiasts.

We expect over 300 participants, with teams of 3-4 members each, which will provide great exposure for Akash Network. This fits well with Akash Network’s focus on innovation and community engagement. Additionally, as I am also working on a cloud gaming startup, your support would benefit both the hackathon and this project.

If you are interested, we can include a special track for projects developed on the Akash Network. It would also be great to have a representative from the Akash Community join us in person. I look forward to discussing how we can build a meaningful partnership.

Thanks again for considering this opportunity.

Best regards,
Abhinav Gupta

**Comment by hewi333** [comment]
Thanks @abhinavcse22 for the quick response. A few more follow up questions to better understand the request:

1. For past hackathons, do you have any links or materials you can share about winning projects or participants?
2. For the 1000 AKT requested sponsorship, what is the breakout of prize funds vs. costs to run the event?
3. I found the website through your X account but don't see anything under [Schedule](https://hackarcode.live/Pages/schedule/schedule.html). Do you have a link to this specific hackathon?
4. What is the focus for this hackathon? i.e. what are you hoping the participants build?
5. Do you have any experience deploying/using Akash Network?
6. Are you okay providing participant lists/emails as part of a sponsorship?

Since there is over a month left, it might be good to socialize this in various community groups as well.

**Reply by abhinavcse22** [reply]
Hey @heyhewi ,

Thank you for your interest and follow-up questions. Please find the details below:

1. You can find the winning projects of our previous hackathons here: 

i)  https://hacknovate5.devfolio.co/projects 
ii) https://buildwithdelhi.devfolio.co/projects

2. The 1000 AKT sponsorship will be divided, with 40% allocated to the prize fund and 60% covering event arrangements, including venue, logistics, and supplies. Your support will significantly enhance the overall experience..

3. The [schedule](https://hackarcode.live/Pages/schedule/schedule.html) is now live on our website, though it is a rough outline at this stage. We will provide a more detailed schedule as we get closer to the event. You can view it along with other details on our [registration page](https://www.hackquest.io/en/hackathon/explore/HackArCode). We are also about to reach 1000 registrations  

4. The focus of this hackathon includes Web2, Web3, cloud computing, and it's also beginner-friendly. We have also reserved 30% of the seats specifically for beginners.

5. Our team has some experience with Akash Network, and one of our organizer working on a cloud gaming project and was considering using Akash Network for it.

6. Yes, we can provide participant lists/emails as part of the sponsorship.

We’ll definitely be promoting this hackathon across various community groups.

Thank you for your support and consideration.

Best regards,
Abhinav Gupta
